test/plugins/graphql.spec.js

```diff
@@ -828,6 +828,26 @@ describe('Plugin', () => {
             .then(() => done())
             .catch(done)
         })
+
+        it('should support multiple executions on a pre-parsed document', () => {
```

Comment:
Should we also be asserting the expected spans are created?

Reply:
It's already asserted by the test for compatibility with Apollo schema stitching.